closure/goog/net/xhrio.js

```diff
@@ -530,8 +539,9 @@ goog.net.XhrIo.prototype.send = function(
 
   // Set up upload/download progress events, if progress events are supported.
   if (this.getProgressEventsEnabled() && 'onprogress' in this.xhr_) {
-    this.xhr_.onprogress =
-        goog.bind(function(e) { this.onProgressHandler_(e, true); }, this);
+    this.xhr_.onprogress = goog.bind(function(e) {
```

Comment:
Na your clang-format seems to be way off. See also CI. Read [Formatting .js with clang-format](https://github.com/google/closure-library/wiki/Formatting-.js-with-clang-format).